src/api/NotificationsAssistant.ts

```diff
@@ -123,6 +138,8 @@ export default class NotificationsAssistant{
 		//Get today's date
 		//want to set today's time to 00H00M00S
 		var dateToday = new Date()
+		dateToday = new Date(dateToday.getFullYear(), dateToday.getMonth(), dateToday.getDate()) 
```

Comment:
Is there a reason why you don't just use new Date() which gives a Date object set to the current date?

https://www.w3schools.com/js/js_dates.asp

Reply:
I think originally, I needed to zero the hour, minute, and second to make the calculation for the times easier, but this was before I discovered iOS only allowed 64 notifications